layouts/index.html

```diff
@@ -132,11 +132,11 @@ <h3 class="subtitle is-3 has-text-centered top-pad">
         {{ partial "js/owlCarousel.html" . }}
 
         {{ if .Site.Params.home.localTime }}
-        {{ $momentjs := resources.Get "/vendor/momentjs/moment.min.js" }}
-        {{ $momentTimezone := resources.Get "/vendor/momentjs/moment-timezone.min.js" }}
-        {{ $momentTimezoneWithData := resources.Get "/vendor/momentjs/moment-timezone-with-data-2012-2022.min.js" }}
-        {{ $initMomentjs := resources.Get "/js/initMoment.js" | resources.ExecuteAsTemplate "js/initMoment.tmp.js" . }}
-        {{ $bundleMoment := slice $momentjs $momentTimezone $momentTimezoneWithData $initMomentjs | resources.Concat "/js/bundleMoment.js" | fingerprint }}
+        {{ $momentjs := resources.Get "vendor/momentjs/moment.min.js" }}
```

Comment:
I double this is part of the problem. But I'm fine with changing it.

Reply:
@hanzei Honestly speaking, I've no idea what Moment JS does.  I'm just replacing everything that starts with a forward slash according to the Hugo experts' recommendations.

Reply:
I was not referring to Moment JS. My point was that I don't think there is a difference between 
```
resources.Get "vendor/some/path"
```
and
```
resources.Get "/vendor/some/path"
```

Reply:
@hanzei Thanks for clarification.  In practice, you're right.  The screenshots that I've taken about my PR for nested Staticman comments were made before this PR.